RESULTS.md

```diff
@@ -318,5 +318,257 @@ We expect to see it go sideways as it adapts to the new proportion of
 resign-disabled games, then show a similar improvement to the v7a improvement.
 After that, if it flattens off again we'll lower the learning rate further.
 
+After our learning rate cuts, the ratings appeared to taper off.  A brief bit of
+"cyclic learning rate" didn't seem to jog it out of its local minima so we
+called a halt after ~530 models
+
+v7 also marked the beginning of an auto-pair for the evaluation matches that
+would attempt to pair the models about whom our rating algorithm had the
```

Comment:
v7 also marked the beginning of an auto-pair for the evaluation matches that
played more games for models with greater rating uncertainty.